client/components/videos-ui/index.jsx

```diff
@@ -97,6 +97,33 @@ const VideosUi = ( {
 		}
 	}, [ course ] );
 
+	let headerTitle;
+	let headerSubtitle;
+	let headerSummary;
+
+	switch ( courseSlug ) {
+		case COURSE_SLUGS.BLOGGING_QUICK_START:
+			headerTitle = translate( 'Watch five videos.' );
+			headerSubtitle = translate( 'Save yourself hours.' );
+			headerSummary = [
+				translate( 'Learn the basics of blogging' ),
+				translate( 'Familiarize yourself with WordPress' ),
+				translate( 'Upskill and save hours' ),
+				translate( 'Set yourself up for blogging success' ),
+			];
+			break;
+		case COURSE_SLUGS.PAYMENTS_FEATURES:
+			headerTitle = translate( 'Add Payments Features' );
+			headerSubtitle = translate( 'Make Money on Your Website' );
+			headerSummary = [
+				translate( 'Making Money with Payments Features' ),
+				translate( 'Premium Membership Blog' ),
+				translate( 'Paid Subscription Newsletter' ),
+				translate( 'Run a Crowdfunding Campaign' ),
+			];
+			break;
```

Comment:
I think this is not intended for this PR, right?

Reply:
Yes, this should go with another PR